python/openvino_tokenizers/convert_tokenizer.py

```diff
@@ -93,9 +94,13 @@ def convert_tokenizer(
         raise OVTypeError(f"Tokenizer type is not supported: {type(tokenizer_object)}")
 
     if isinstance(ov_tokenizers, tuple):
+        # import openvino as ov
+        # ov.runtime.save_model(ov_tokenizers[1], "openvino_detokenizer.xml")
+
```

Comment:
Fix

Reply:
my bad 🤦in the very last moment suddenly pushed this. Will remove